thoth/user_api/openapi_server.py

```diff
@@ -79,6 +79,20 @@
 
 # static information as metric
 metrics.info("user_api_info", "User API info", version=__version__)
+_API_GAUGE_METRIC = metrics.info("user_api_schema_up2date", "User API schema up2date")
+
+
+@application.before_request
+def before_request_callback():
+    """Callback registered, runs before each request to this service."""
+    method = request.method
+    path = request.path
+
+    # Update up2date metric exposed.
+    if method == "GET" and path == "/metrics":
+        graph = GraphDatabase()
+        graph.connect()
+        _API_GAUGE_METRIC.set(int(graph.is_schema_up2date()))
```

Comment:
We can also do `_API_GAUGE_METRIC.set(1 - int(graph.is_schema_up2date()))`. This way we will have `1` signalizing an issue, `0` signalizing OK state. We should have this consistent across metrics.

Reply:
@fridex That is what we have in connection errors in `metrics-exporter` for Database and Ceph, but I feel like, it should be as you said actually, like `1` everything is working, so active and `0` when something is wrong and it is not working.